### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by bobkelso974** [sblantipodi/solar_station#1]
Hello
I'm interested in your project 
I'd like to ask you a few questions:
What model of water pump do you use?
How long do you run this pump? (deep sleep time?)
Is it for indoor or outdoor use?
And finally, does it work well?

I made about the same circuit.
Except I didn't put a relay but a Mosfet for energy saving.

Thank you

**Comment by sblantipodi** [comment]
@bobkelso974 sorry for the late reply.
I'm using a 3.5V-9V 1-3W water pump.

I have added an image of the pump in the readme of the project.
https://github.com/sblantipodi/solar_station

You can find similar pump on amazon, it works very very well. 
I run it at 8.6V so a little bit less than 3W and it is very powerful and quiet.

Pump runs for 30 seconds a day because I use it for a small watering system.
Take a look at the readme, there is an IP56 box for outdoor use.
The ESP sleeps for 10 minutes and it stays on the time needed to read the configuration from the MQTT server and send back its state.
Basically when the ESP wake up reads the configuration from the MQTT server, this config says to the ESP if the pump must be on and the number of seconds that it must be turned on.

I can say that the system works very very well.

**Comment by bobkelso974** [comment]
Thank you for your answers.
I have the same model of water pump as you but I have a problem, I have almost the same circuit as you expected the relay is replaced by a mosfet
The problem is on my mt3608. When I set the voltage to 8v, I have a hiss on this module and when the pump is running the voltage goes up to 10v ...
When I test with a mini USB boost converter 3V-5V, it works well. I tested several MT3608 same problem
Did you have the same type of problem? Or do you have an idea where it can come from?

**Comment by sblantipodi** [comment]
mmm...
A hiss is not necessary a problem, it could be the normal "coil whine" but it's not normal that if you set the MT3608 to output 8V it output 10V.

Is your MT3608 connected to the battery directly like in my circuit?
If your battery able to handle the load of the pump? 
Battery tends to have a big voltage drop when something like a motor is connected, after the motor startup the voltage returns to normal and this may cause some spikes.
A good capacitor can help with this problem or probably try to choose a better 18650 battery rated for high drain. I know that 3W is not that much for a 18650 (I hope you are using a 18650) but motors are a bit tricky because they cause huge voltage drops and spikes on power supply.

**Comment by sblantipodi** [comment]
@bobkelso974 another reason why I choosed the relay is because relays provide isolation between the command circuit, this isolation is a good thing when using motors.

Mosfets have a "body diode" and the body is normally connected to the source and this is not desirable when using motors or things that can provide some big inrush current in your circuit

**Comment by bobkelso974** [comment]
thank you for all your answers
I'm going to do some tests, including a relay to replace the mosfet...
